### PR TITLE
fix marks only one bug

### PR DESCRIPTION
**Pull request by maolion**

### CODE REVIEW

**Comment by benjycui** [comment]
Could you fix those ESLint errors? And add a unit test for one mark.

File: src/Marks.jsx [hunk 1]
```diff
@@ -4,7 +4,7 @@ import classNames from 'classnames';
 const Marks = ({className, marks, included, upperBound, lowerBound, max, min}) => {
   const marksKeys = Object.keys(marks);
   const marksCount = marksKeys.length;
-  const unit = 100 / (marksCount - 1);
+  const unit = 100 / ((marksCount - 1)||1);
```
Comment (just-boris) [inline src/Marks.jsx]: Maybe `Math.min(marksCount - 1, 1)`

Comment (benjycui) [inline src/Marks.jsx]: `Math.min(1 - 1, 1)` => 0

Comment (just-boris) [inline src/Marks.jsx]: I meant `Math.max(marksCount - 1, 1)`, sorry

**Comment by benjycui** [comment]
Close for no response for a long time.

**Pull request closed without merging**